spec/file-spec.coffee

```diff
@@ -3,6 +3,7 @@ fs = require 'fs-plus'
 temp = require 'temp'
 File = require '../lib/file'
 PathWatcher = require '../lib/main'
+Q = require 'q'
```

Comment:
Isn't Atom transitioning to native promises now?  Or is that just for Atom packages and not Node ones?